app/components/ui/Navigation/DesktopMenuItems.tsx

```diff
@@ -41,7 +43,10 @@ const DesktopMenuItems = ({
     <Link
       key={uniqueKey}
       to={menuItem.url}
-      className={styles.navigationMenuLink}
+      className={classNames(
+        dropdownMenuStyles.menuLink,
+        desktopNavigationStyles.navigationMenuLink
+      )}
```

Comment:
I'm trying to pull apart styling that just affects the header nav vs the reusable `DropdownMenu` component

Reply:
I'm confused on this. DropdownMenu gets all its styles from DropdownMenu.module.scss so I think DesktopMenuItems should get all its styles from DesktopNavigation.module.scss (though we can change it to DesktopMenuItems.module.scss if we want to)

Reply:
Good catch, it's actually redundant to have both these classes.